### PR TITLE
Add resize of Span Detail drawer

### LINKED ISSUE #1216: [Implementation] Resizing size of span detail attributes area
[**Describe the enhancement you'd like to see**
Implementation ticket to add resizing feature to the attributes drawer

**Additional context**
Add any other context or screenshots about the feature request here.](https://github.com/kubeshop/tracetest/issues/1153)

### PR DESCRIPTION
**Pull request by jorgeepc**
This PR implements the resize feature for the Span Detail drawer.

## Changes

- Drawer layout

## Fixes

- fixes #1216

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Loom

https://www.loom.com/share/b86c878ff90d426c9f45813ef07ed68b

### CODE REVIEW

**Review by cescoferraro — APPROVED** [review]
LGTM! Thanks Jorge

File: web/package.json [hunk 1]
```diff
@@ -70,7 +70,7 @@
     "types:generate": "openapi-typescript ../api/openapi.yaml --output ./src/types/Generated.types.ts",
     "grammar:generate": "lezer-generator ./src/utils/grammar/syntax.grammar -o ./src/utils/grammar/grammar.js",
     "types": "tsc",
-    "eslint": "eslint --ext=.js,.ts,.tsx ./src/**/* ./cypress/integration/**/*",
+    "eslint": "eslint --ext=.js,.ts,.tsx ./src/**/* ./cypress/e2e/**/*",
```
Comment (xoscar) [inline web/package.json:73]: Good catch

File: web/src/components/Drawer/Drawer.styled.ts [hunk 2]
```diff
@@ -1,40 +1,37 @@
-import styled, {css} from 'styled-components';
+import styled, {createGlobalStyle, css} from 'styled-components';
+
+export const GlobalStyle = createGlobalStyle`
+  .reflex-container.vertical > .reflex-splitter {
```
Comment (xoscar) [inline web/src/components/Drawer/Drawer.styled.ts:4]: Isn't this at the level where we can use our theme?

**Review by xoscar — APPROVED** [review]
LGTM!

**Pull request merged**